### ISSUE TITLE
Portable version?

### ISSUE THREAD

**Issue by AlanEdd** [bruhov/WinThumbsPreloader#32]
Is there any way to use this as a portable version ?
I am trying to but when it runs on command line... the welcome screen is just popping up. I need to run it as non admin without installing.
Possible ?

**Comment by VL4DST3R** [comment]
I've used this as if it were portable for a long time and it worked just fine, however I would suggest trying the updated fork [here](https://github.com/Mfarooq360/WinThumbsPreloader-V2).

**Comment by AlanEdd** [comment]
Thank you for that, no the standalone from the fork doesn't work for me either, I left a message on the updated fork.
Thanks for the reply.

**Comment by AlanEdd** [comment]
Yes! that seems to work, thank you so much. 
[path to winthumbspreloader.exe can't have non standard characters]

As a general question about this software: It preloads the thumbnails, does that mean that it loads them into RAM for the session and that it is flushed on restarting the machine ?
I see how it works but don't fully understand what is happening behind the scenes... 

Thank you ! 
Alan.

**Comment by VL4DST3R** [comment]
> As a general question about this software: It preloads the thumbnails, does that mean that it loads them into RAM for the session and that it is flushed on restarting the machine ?

It calls the windows API to generate cached thumbnails in the same manner you can generate them yourself by opening a folder inside file explorer with images or video and letting their thumbnails load up naturally. 

They all get saved inside a series of databases stored in your users folder (on your drive), and do not get removed unless windows itself decides (for one of a plethora of annoying reasons) to erase them and rebuild them from scratch, which can happen from anything from running low on disk space to crashes, windows updates or even restarts.

**Comment by AlanEdd** [comment]
Ah so it builds up the Windows Thumbnail cache then... 
%LocalAppData%\Microsoft\Windows\Explorer\
Is that correct ?

**Comment by VL4DST3R** [comment]
Precisely. Before the V2 fork added features to backup and restore those files inside the app UI itself, I used to copy and restore them manually after a wipe.

**Comment by freemedom** [comment]
AppData\Local\Microsoft\Windows\Explorer\thumbcache_xx.db
